File: QuizApp/src/components/Quiz.js
import "./Quiz.css";
function Quiz()
{
    var quiz={
        title:"fun with maths",
        description:"This quiz is just to have fun with maths",
        category:"Maths"
    }
    return(
        <div className="quiz">
            <div>
                Quiz Title:{quiz.title}
                <br/>
                Quiz Description:{quiz.description}
                <br/>
                Quiz Category:{quiz.category}
            </div>
        </div>
    )
}
export default Quiz;

File: QuizApp/Models/Quiz.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace QuizApp.Models
{
    public class Quiz
    {
        [Key]
        public int QuizId { get; set; }

        [Required(ErrorMessage = "Title of the Quiz cannot be empty")]
        public string Title { get; set; }
        public string Description { get; set; }
        public string Category { get;  set; }
        public int? TimeLimit { get; set; }
        [JsonIgnore]
        public ICollection<Questions>? Questions { get; set; }
        
    }
}